tests/testthat/test-extract_ngram.R

```diff
@@ -0,0 +1,22 @@
+# Test for function extract_ngram
+helper_data <- c("Make", "America", "Great", "Again", "DonaldTrump")
+test_that("Generated ngrams is incorrect", {
+  expect_equal(extract_ngram(helper_data, n=1), c("Make", "America", "Great", "Again", "DonaldTrump"))
+})
+test_that("Generated ngrams is incorrect", {
+  expect_equal(extract_ngram(helper_data, n=3), c("Make America Great", "America Great Again", "Great Again DonaldTrump"))
+})
+test_that("Generated ngrams is incorrect", {
+  expect_equal(extract_ngram(helper_data, n=4), c("Make America Great Again", "America Great Again DonaldTrump"))
+})
+test_that("Generated ngrams is incorrect", {
+  expect_equal(extract_ngram(helper_data, n=5), c("Make America Great Again DonaldTrump"))
+})
```

Comment:
I think the code will look cleaner if we group similar expectations under the same `test_that`.

```suggestion
test_that("Generated ngrams is incorrect", {
  expect_equal(extract_ngram(helper_data, n=1), c("Make", "America", "Great", "Again", "DonaldTrump"))
  expect_equal(extract_ngram(helper_data, n=3), c("Make America Great", "America Great Again", "Great Again DonaldTrump"))
  expect_equal(extract_ngram(helper_data, n=4), c("Make America Great Again", "America Great Again DonaldTrump"))
  expect_equal(extract_ngram(helper_data, n=5), c("Make America Great Again DonaldTrump"))
```